app/core/services/PageBuilder.scala

```diff
@@ -105,7 +106,9 @@ class PageBuilder @Inject() (val placeholders: Placeholders) extends ProcessPopu
           f(page.id, page.url, text.english)
         case i: Question =>
           f(page.id, page.url, hintRegex.replaceAllIn(i.text.english, ""))
-        case i: Sequence =>
+        case i: NonExclusiveSequence =>
```

Comment:
one Sequence trait match here could replace matching on NonExclusiveSequence and ExclusiveSequence

Reply:
Done